frontend/talentsearch/resources/js/components/ExperienceAccordion/ExperienceAccordion.tsx

```diff
@@ -0,0 +1,59 @@
+// Using graphql __TypeName property for type guard discriminator
+/* eslint-disable no-underscore-dangle */
+import React from "react";
+import { Accordion } from "@common/components/accordion/Accordion";
+import {
+  AwardExperience,
+  CommunityExperience,
+  EducationExperience,
+  PersonalExperience,
+  WorkExperience,
+} from "../../api/generated";
+import AwardAccordion from "./individualExperienceAccordions/AwardAccordion";
+import CommunityAccordion from "./individualExperienceAccordions/CommunityAccordion";
+import EducationAccordion from "./individualExperienceAccordions/EducationAccordion";
+import PersonalAccordion from "./individualExperienceAccordions/PersonalAccordion";
+import WorkAccordion from "./individualExperienceAccordions/WorkAccordion";
+
+type AnyExperience =
```

Comment:
does it have to be AnyExperience ? Why not just "Experience"

Reply:
Experience already exists, so it would be confusing to reuse it
https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/api/graphql/schema.graphql#L278